bin/multitest

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "weaviate"
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+# require "pry"
+# Pry.start
+
+client = Weaviate::Client.new(
+  url: ENV["WEAVIATE_URL"],
+  api_key: ENV["WEAVIATE_API_KEY"],
+  model_service: :openai,
+  model_service_api_key: ENV["MODEL_SERVICE_API_KEY"]
+)
+
+puts "Creating a tenant"
+WeaviateClient.client.schema.create(
```

Comment:
I suspect this is for some sort of a internal test you were running. Could you please delete this file?